codewof/programming/content/en/driver-speed/question.md

```diff
@@ -0,0 +1,6 @@
+# Driver speed
+
+Write a function `check_speed(speed)` that checks the speed of drivers. If the speed is less than or equal to 50, it should **print** `Ok.`.
+Otherwise, for every 5km above the speed limit (50), it should give the driver one demerit point and print the total number of demerit points. If the driver gets more than 5 points the function should print `License suspended.`. 
```

Comment:
Also what happens between 50 and 55? Sounds like it should print 0

Reply:
Updated. Also in regards to the between 50 and 55 scenario there is an example for 53 at the end of the question showing that it should print 0.

Reply:
I've changed this question now so instead of calculating the number of demerit points it just checks to see if they are going to fast, and then checks if they are going so fast that they get their license suspended.